libs/sdk-core/src/models.rs

```diff
@@ -805,6 +829,27 @@ pub struct SweepResponse {
     pub txid: Vec<u8>,
 }
 
+pub struct SendOnchainRequest {
+    pub amount_sat: u64,
+    pub onchain_recipient_address: String,
+    pub pair_hash: String,
+    pub sat_per_vbyte: u64,
```

Comment:
Here `sat_per_vbyte` is an `u64` but in `RefundRequest` it's an `u32`.

For consistency `u32` could be used for both.

Reply:
Done c579ec0da3c0d64d74b2b64bdfb334a81dc01b9f